src/main/java/org/folio/oaipmh/dao/impl/SetDaoImpl.java

```diff
@@ -99,18 +105,19 @@ public SetDaoImpl(final PostgresClientFactory postgresClientFactory) {
 
   private Future<FolioSet> saveSetItem(FolioSet entry, String tenantId, String userId) {
     prepareSetMetadata(entry, userId, InsertType.INSERT);
-    return getQueryExecutor(tenantId).transaction(queryExecutor -> queryExecutor.executeAny(dslContext -> dslContext.insertInto(SET)
-      .set(toDatabaseSetRecord(entry))
-      .onConflict(SET.ID)
-      .doNothing()
-      .returning())
+    return getQueryExecutor(tenantId).transaction(queryExecutor -> queryExecutor
+      .executeAny(dslContext -> dslContext.insertInto(SET_LB)
+        .set(toDatabaseSetRecord(entry))
+        .onConflict(SET_LB.ID)
+        .doNothing()
```

Comment:
I'm not familiar with this client... if there's a conflict I see it's not overwriting but doing nothing, which makes sense.  I'm wondering what the caller would see... Do they get an error response or would it just fail silently?

Reply:
"onConflict(SET_LB.ID).doNothing" actually is redundant here cause validation on already existed id happens before invoking saveSetItem method. I'm going to remove it now.